docs/JestObjectAPI.md

```diff
@@ -106,6 +169,32 @@ mocked version of the module for you.
 This is useful when you want to create a [manual mock](ManualMocks.md) that
 extends the automatic mock's behavior.
 
+Example:
+
+```js
+// utils.js
+export default {
+  authorize: () => {
+    // implementation
+    return 'token';
+  },
+  isAuthorized: secret => secret === 'wizard',
+};
+```
+
+```js
+// __tests__/genMockFromModule.test.js
+import utils from '../utils';
```

Comment:
Please remove this, as it's not necessary and redefining it below will throw error.